catalog/app/components/Assistant/UI/Chat/DevTools.tsx

```diff
@@ -1,11 +1,144 @@
 import * as Eff from 'effect'
 import * as React from 'react'
 import * as M from '@material-ui/core'
+import ClearIcon from '@material-ui/icons/Clear'
+import DeleteIcon from '@material-ui/icons/Delete'
+import GetAppIcon from '@material-ui/icons/GetApp'
```

Comment:
I'd prefer this variant for consistency

```ts
import { Clear as IconClear, Delete as IconDelete, GetApp as IconGetApp } from '@material-ui/icons'
```

Reply:
wildcard-import makes it more difficult to control which icons we use

Reply:
I'm ok with `import * as Icons`, though. If Icons is a component in our codebase.
Like this https://github.com/quiltdata/quilt/blob/4d5d6cb9b86beb8da447eaa1191cb4c5bccaf1e3/catalog/app/components/Icon.tsx

Reply:
> I'm ok with import * as Icons, though. If Icons is a component in our codebase.

that would require maintaining our subset of icons and remembering to review and prune it continuously, which seems like a maintenance burden with no clear benefit